### PR TITLE
Clarify Prometheus TLS settings.

### PR DESCRIPTION
**Pull request by bianpengyuan**
https://github.com/istio/istio/issues/29919

The doc is a bit confusing and the order of configuration is wrong.

### CODE REVIEW

**Review by douglas-reid — APPROVED** [review]

File: content/en/docs/ops/integrations/prometheus/index.md [hunk 1]
```diff
@@ -83,30 +83,38 @@ The control plane, gateway, and Envoy sidecar metrics will all be scraped over p
 One way to provision Istio certificates for Prometheus is by injecting a sidecar which will rotate SDS certificates and output them to a volume that can be shared with Prometheus.
 However, the sidecar should not intercept requests for Prometheus because the Prometheus's model of direct endpoint access is incompatible with Istio's sidecar proxy model.
 
-Add the following annotations to the Prometheus deployment to inject a sidecar that will write a certificate to a shared volume, but without configuring traffic redirection:
+To achieve this, configure a cert volume mount at Prometheus server container:
```
Comment (ericvn) [inline content/en/docs/ops/integrations/prometheus/index.md:86]:
```suggestion
To achieve this, configure a cert volume mount on the Prometheus server container:
```

File: content/en/docs/ops/integrations/prometheus/index.md [hunk 2]
```diff
@@ -83,30 +83,38 @@ The control plane, gateway, and Envoy sidecar metrics will all be scraped over p
...
+    volumeMounts:
+      mountPath: /etc/prom-certs/
+      name: istio-certs
+volumes:
+  - emptyDir:
+      medium: Memory
+    name: istio-certs
+{{< /text >}}
+
+Then add the following annotations to the Prometheus deployment pod template, and deploy it with [sidecar injected](/docs/setup/additional-setup/sidecar-injection/).
```
Comment (ericvn) [inline content/en/docs/ops/integrations/prometheus/index.md:101]:
```suggestion
Then add the following annotations to the Prometheus deployment pod template, and deploy it with [sidecar injection](/docs/setup/additional-setup/sidecar-injection/).
```

File: content/en/docs/ops/integrations/prometheus/index.md [hunk 3]
```diff
@@ -83,30 +83,38 @@ The control plane, gateway, and Envoy sidecar metrics will all be scraped over p
...
 {{< /text >}}
 
-To use the provisioned certificate, mount the shared volume for the Prometheus container and set the scraping job TLS context as follow:
-
-{{< text yaml >}}
-volumeMounts:
-- mountPath: /etc/prom-certs/
-  name: istio-certs
-{{< /text >}}
+Finally, set the scraping job TLS context as follow:
```
Comment (ericvn) [inline content/en/docs/ops/integrations/prometheus/index.md:117]:
```suggestion
Finally, set the scraping job TLS context as follows:
```

**Comment by ericvn** [comment]
Should this PR also be cherrypicked to release-1.9?

**Review by ericvn — APPROVED** [review]

**Comment by bianpengyuan** [comment]
@ericvn yep, thanks!

**Pull request merged**